rules/misc_rules.build_defs

```diff
@@ -363,7 +363,7 @@ def remote_file(name:str, url:str|list, hashes:list=None, out:str=None, binary:b
             labels = labels,
         )
         if strip_prefix:
-            cmd = '"$TOOL" x $SRCS -o "$OUT" -s ' + strip_prefix
```

Comment:
What's happened here? Do you not need the OUT file name?

Reply:
Ah that's where that went... I thought it was on another branch.

I'll split it out to a separate PR. I don't think the current setup works basically (at least I can't make it do anything I want...)